less/v2/inventory.less

```diff
@@ -464,17 +468,38 @@
         align-items: center;
       }
 
-      @container (min-width: 600px) {
-        .item-price, .item-formula, .spell-uses { display: flex; }
+      /* Item Description */
+      .item-description {
+        font-size: var(--font-size-11);
+
+        .item-summary {
```

Comment:
I've noticed that if you don't have a description then the pills get push right up against the rest of the row:

<img width="184" alt="Screenshot 2024-07-04 at 09 19 27" src="https://github.com/foundryvtt/dnd5e/assets/19979839/cc7724f2-aff9-4112-b3b9-ca1a1bbdd95c">

Probably want a bit of padding there to make it a bit more even compared to the bottom padding.